packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -32,25 +32,26 @@ Three Options are possible:
     - Both: clicking within and outside the dropdown closes the dropdown.
 - **Placement:** Defines where the dropdown appears when it is opened. The position depends on the element that opens the dropdown (e.g. button). By default, the dropdown is displayed at the bottom right of the button. 
 You can choose between different directions (top, bottom, left, right) and two options for the alignment on the button (start, end). When there is not enough space for the selected placement, it will be corrected automatically. The placement of the submenu is always generated automatically. We typically use the default placement option to ensure consistency. 
-- **Date Selection:** Use the element [Date dropdown](date-dropdown.md) to get a date selection in the dropdown.
+- **Date Selection:** Use the component [date dropdown](date-dropdown.md) to get a date selection in the dropdown.
 
 ![Dropdown Examples](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2372-2696&mode=design&t=OVHeXvLZYLkP2CzN-4)
 
 ## Behavior in context
-- **Text truncation:**  Item and header labels are not truncated. The whole text in the items and headers consists of one line only. The text is cut off after 58 characters with "...".
+- **Text truncation:**  Item and header labels are not truncated. The whole text in the items and headers consists of one line only. The text is cut off after 58 characters.
```

Comment:
Stay consistent in wording: label instead of text, and I would use "truncated" also for the last sentence, since it is actually was happening